nornir_nautobot/plugins/inventory/nautobot.py

```diff
@@ -32,15 +32,16 @@ def _set_host(data: Dict[str, Any], name: str, groups, host) -> Host:
             port=value.get("port"),
             username=value.get("username"),
             password=value.get("password"),
-            platform=host.get("platform", {}).get("slug"),
+            platform=host.get("data", {}).get("pynautobot_dictionary", {}).get("platform", {}).get("slug"),
             extras=value.get("extras"),
         )
+
     return Host(
         name=name,
         hostname=host["hostname"],
         username=host.get("username"),
         password=host.get("password"),
-        platform=host.get("platform", {}).get("slug"),
+        platform=host.get("data", {}).get("pynautobot_dictionary", {}).get("platform", {}).get("slug"),
```

Comment:
Would it make sense set platform within the `load` method instead traversing the dictionary again and it would also simplify L35 to not have to traverse in as well.

Reply:
@FragmentedPacket let me know if that is what you were thinking on this.

Reply:
It's pretty close to what I was thinking, but it definitely works!